### DISCUSSION TITLE
NumericPicker doesnt apply readonly attribute

### DISCUSSION THREAD

**Discussion by yangetze** [Megabit/Blazorise#4418] [Q&A]
I have this 

`<NumericPicker @bind-Value="@_model.Amount" Placeholder="Amount"  Min="0" Max="999999999999" Step="1" Decimals="0" EnableStep="false" ReadOnly />`

But, when I run the project, this is the html result 
![image](https://user-images.githubusercontent.com/23171333/209715708-6018611e-092e-43ed-aefd-ad8d2ed5a14c.png)
Doesn't write Readonly attributte. 

But if I use Disabled attribute works perfectly.

Blazorise v.1.1.4.1

**Comment by stsrki** [comment]
Seems like a bug. I will open a new issue to fix it. Thanks for letting us know.
